packages/atomic/src/components/result-template-components/atomic-result-fields-list/atomic-result-fields-list.tsx

```diff
@@ -13,13 +13,12 @@ export class AtomicResultFieldsList {
   @ResultContext()
   private resizeObserver!: ResizeObserver;
 
-  private updatingChildren = false;
```

Comment:
this was failing cypress tests. After making those changes, tests passed once, and I'm not sure yet if it's a fluke.